jerry-core/ecma/base/ecma-globals.h

```diff
@@ -79,6 +79,7 @@ typedef enum
   ECMA_SIMPLE_VALUE_UNDEFINED, /**< undefined value */
   ECMA_SIMPLE_VALUE_NULL, /**< null value */
   ECMA_SIMPLE_VALUE_NOT_FOUND, /**< a special value returned by ecma_op_object_find */
+  ECMA_SIMPLE_VALUE_NOT_FOUND_AND_STOP, /** not found and not search the proto chain */
```

Comment:
Why don't we return with (virtual) undefined in this case? Is there a situation where we need to know that the value is not found?